docs/Node Operators/evm-performance-configuration.md

```diff
@@ -202,6 +202,9 @@ RPCGasCap = 0
 RPCTxFeeCap = 0.0
 ```
 
+> 🚧 If you are using `Nethermind` instead of `go-ethereum` you should be aware that the `gascap` flag behaves differently, resulting in errors like [this one](https://github.com/NethermindEth/nethermind/issues/4307).
+> In this case, just set `gascap` to a high number like `5000000000` or just don't set it at all, to avoid issues.
```

Comment:
```suggestion
> 🚧 If you are using `Nethermind` instead of `go-ethereum` you should be aware that the `gascap` flag behaves differently, resulting in errors like [this one](https://github.com/NethermindEth/nethermind/issues/4307).
> In this case, you can set `gascap` to a very large number like `5000000000`.
```